src/compas_rhino/scene/brepobject.py

```diff
@@ -12,17 +12,7 @@
 
 
 class RhinoBrepObject(RhinoSceneObject, GeometryObject):
-    """A scene object for drawing a RhinoBrep.
-
-    Parameters
-    ----------
-    brep : :class:`compas_rhino.geometry.RhinoBrep`
-        The Brep to draw.
-
-    """
-
-    def __init__(self, brep, **kwargs):
-        super(RhinoBrepObject, self).__init__(geometry=brep, **kwargs)
+    """A scene object for drawing a RhinoBrep."""
```

Comment:
In many cases the `__init__` function of subclass can be entirely omitted, because the don't do anything else that passing in the positional argument using a different name.